friends/src/Components/FriendList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {connect} from 'react-redux';
+
+import Friend from "./Friend";
+import {getFriends} from "../Actions"
+
+
+class FriendList extends React.Component{
+    state={
+        friends:[],
+        loadingFriends:[],
+    }
+
+    componentDidMout(){
+        this.props.getFriends();
+    }
+
+
+    render(){
+        return (
+            <div>
+                {this.props.friends.map(friend=>(
+                    <Friend friend={friend}/>
+                ))}
```

Comment:
Make sure to implement conditional rendering when mapping through the `this.props.friends` array.

By doing this:
```
{this.props.friends && this.props.friends.map(friend=>(
    <Friend friend={friend}/>
  )
)}
```

You will ensure (with the `&&` operator - for short-circuit AND operation) that the `map` method will run only if `this.props.friends` actually has a truthy value (not `null` or `undefined`, which may crash the app).